src/test/java/io/jenkins/plugins/opentelemetry/LifecycleDebugger.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
A test class?

Reply:
yes to understand the lifecycle. Note that there is a bug in `io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule`, it's not the same lifecycle, the JCasC configuration happens after `hudson.init.InitMilestone#COMPLETED` when it happens on `hudson.init.InitMilestone#SYSTEM_CONFIG_ADAPTED` in real life.